test/api-digester/source-stability.swift.expected

```diff
@@ -1,4 +1,6 @@
 /* FIXME: Bogus */
+Struct DictionaryIterator has generic signature change from <Key, Value where Key : Hashable> to <Element>
+Struct SetIterator has generic signature change from <Element where Element : Hashable> to <Key, Value>
```

Comment:
Wait, this seems backwards? Why would they switch like this?

Reply:
The correct diags are actually like these:
-Struct DictionaryIterator has generic signature change from <Key, Value where Key : Hashable> to <Element>
-Struct SetIterator has generic signature change from <Element where Element : Hashable> to <Key, Value>
+Struct DictionaryIterator has generic signature change from <Key, Value where Key : Hashable> to <Key, Value>
+Struct SetIterator has generic signature change from <Element where Element : Hashable> to <Element>

Reply:
I copy & pasted the ones in this PR from my own test logs, with no edits -- is it possible the api-digester output is somehow nondeterministic?